challenges/cloud/sre-server.go

```diff
@@ -0,0 +1,65 @@
+package main
```

Comment:
@jimbishopp will candidates fork this repo, and extend it? Or just copy & paste?

Reply:
I changed the link so it points to the go file. Candidates can click the link and download it. Forking isn't necessary.